packages/grid/x-data-grid-pro/src/tests/clipboard.DataGridPro.test.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
 import { GridApi, useGridApiRef, DataGridPro, DataGridProProps } from '@mui/x-data-grid-pro';
+// @ts-ignore
 import { createRenderer, fireEvent } from '@mui/monorepo/test/utils';
 import { expect } from 'chai';
-import { stub } from 'sinon';
+import Sinon, { stub } from 'sinon';
```

Comment:
```suggestion
import { SinonStub, stub } from 'sinon';
```

or put everything on the default export

```suggestion
import * as Sinon from 'sinon';
```